src/wasm-traversal.h

```diff
@@ -49,6 +49,7 @@ struct Visitor {
   ReturnType visitSetGlobal(SetGlobal* curr) {}
   ReturnType visitLoad(Load* curr) {}
   ReturnType visitStore(Store* curr) {}
+  ReturnType visitAtomicRMW(AtomicRMW* curr) {return ReturnType();} //Stub impl so not every pass has to implement this yet.
```

Comment:
I'm ok with landing this PR with not all opt passes working yet, we can fix those later. Could we keep this line normal-looking, then, with `{}` like the others? Or is the return here temporarily necessary?